get_started/what_is_citus.rst

```diff
@@ -66,7 +44,31 @@ For more customers and statistics, see our `customer stories <https://www.citusd
 When to Use Citus
 =================
 
-Citus serves many use cases. Two common ones are scaling multi-tenant (B2B) databases and real-time analytics. In addition to the information below, there are examples of Citus `use-cases <https://www.citusdata.com/use-cases>`_ and `customer case studies <https://www.citusdata.com/solutions/case-studies>`_ on our main web site.
+Citus serves many use cases. Two common ones are scaling multi-tenant (B2B) databases and real-time analytics.
+
+Fast-growing `multi-tenant apps <https://www.citusdata.com/use-cases/multi-tenant-apps>`_ want to add new customers, deliver great performance, and not have to worry about database infrastructure. Data analysts want sub-second response times for customer-facing `analytics dashboards <https://www.citusdata.com/use-cases/real-time-analytics>`_, even with real-time ingestion, very large data sets, complex queries, and lots of concurrent users.
+
+Citus allows these and other applications to enjoy the power and familiarity of a traditional relational database, but with the capability for massive scale. Applications connect to and use a Citus-enabled PostgreSQL database just like they would a traditional PostgreSQL database. Citus uses the :ref:`same SQL commands <citus_sql_reference>` that developers and frameworks already know.
+
++--------------------------------------------------------+-------------------------------------------------------------+
+| Multi-Tenant Advantages                                | Real-Time Advantages                                        |
```

Comment:
I'm now wondering if we just fold in these advantages as bullet points in the below subsections. I recall I had recommended the table approach, but this seems like it compares one vs the other, which isn't quite what we want. 
Could you make that change @begriffs ?

Reply:
I see what you mean about the comparison feeling -- and that a user doesn't look at the advantages of each in order to determine which to use, but rather matches their modeling needs. I do like how clear the advantages in the table are at a glance, so I'll convert the table rows to bullet points like you suggest.